interview-questions.md

```diff
@@ -7,52 +7,58 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What are props in React?
 
-  Your answer:
+  Your answer: Props is short for properties in React. It is a way that we can pass data from one component to another. For example, if we have a component instantiated in the main app.js file. We could give it a property of "name" and set it to value. That value could be used in the component file by using props.name .
 
-  Researched answer:
+  <Example 
+    name = "sample"
+  />
+
+  props.name = "sample" in the component the props would be passed onto.
+
+  Researched answer: Props, or properties, are  used to pass data from one component to another. More so, it allows for that data to be dynamic when being passed between components. It allows you to change data in different components without having to rewrite the code in each component.
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event is a change that can be caused by user input or a click for example. When a user clicks on a certain part of the application, we might expect something to occur. This is called an onclick and is an example of a DOM event. 
 
-  Researched answer:
+  Researched answer: An event that causes a change in the DOM which is done through an event handler. The event handler is connected to a certain function in the code and an example would be an onclick, as mentioned before. The click will trigger a piece of code to run when the mouse is clicked in that component that has that onclick attribute. 
```

Comment:
The DOM is a visual representation of the markup and code logic that makes up a website.  It is important to note the difference between this and the Virtual DOM which listens for any diff created by user interactions.